digdag-standards/src/main/java/io/digdag/standards/operator/HttpOperatorFactory.java

```diff
@@ -177,13 +180,36 @@ else if (user.isPresent()) {
                 request.header(header.getKey(), header.getValue());
             }
 
+            configureQueryParameters(request);
+
             ContentResponse response = PollingRetryExecutor.pollingRetryExecutor(state, "request")
                     .withErrorMessage("HTTP request failed")
                     .run(s -> execute(request));
 
             return result(response, storeContent);
         }
 
+        private void configureQueryParameters(Request request)
+        {
+            // TODO: allow secret query parameters
+
+            try {
+                Config queryParameters = params.parseNestedOrGetEmpty("query");
```

Comment:
I see > https://tools.ietf.org/html/rfc3986#section-3

The second one seems clearer to me since I guess `query_string` is more familiar. But either might be fine.